dirbot/spiders/dmoz.py

```diff
@@ -20,15 +20,17 @@ def parse(self, response):
         @url http://www.dmoz.org/Computers/Programming/Languages/Python/Resources/
         @scrapes name
         """
-        sel = Selector(response)
-        sites = sel.xpath('//ul[@class="directory-url"]/li')
+        sites = response.css('#site-list-content > div.site-item > div.title-and-desc')
         items = []
 
         for site in sites:
             item = Website()
-            item['name'] = site.xpath('a/text()').extract()
-            item['url'] = site.xpath('a/@href').extract()
-            item['description'] = site.xpath('text()').re('-\s[^\n]*\\r')
+            item['name'] = site.css(
```

Comment:
Why not leave both the xpath and css examples? Tutorial is using xpath and css for the last part.